### PR TITLE
Linkify and truncate links in announcements

### PR DESCRIPTION
**Pull request by ollfkaih**
Before
image
After
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +60,8 @@
     "jsdom": "^20.0.0",
     "jszip": "^3.10.1",
     "jwt-decode": "3.1.2",
+    "linkify-react": "^3.0.4",
+    "linkifyjs": "^3.0.5",
```
Comment (ivarnakken) [inline package.json:64]: Is `linkifyjs` also needed?

**Review by ivarnakken — APPROVED** [review]
Looks very nice! 💯

File: package.json (hunk 1 above)
Comment (LudvigHz) [reply package.json:64]: It's a peer-dependency, so unfortunately yes :'(

**Review by LudvigHz — APPROVED** [review]
Nice :100: 

It would be awesome if you could add this to polls (description) as well :grimacing:

**Comment by ivarnakken** [comment]
You'll have to lint your new changes. Please just amend it 😄

**Pull request merged**